taqueria-vscode-extension/src/lib/helpers.ts

```diff
@@ -740,6 +762,52 @@ export class VsCodeHelper {
 		);
 	}
 
+	exposeTestSetupCommand(projectDir?: Util.PathToDir) {
+		this.registerCommand(
+			COMMAND_PREFIX + 'create_test_folder',
+			async () => {
+				const folders = await this.vscode.window.showOpenDialog({
+					canSelectFiles: false,
+					canSelectFolders: true,
+					canSelectMany: false,
+					openLabel: 'Select',
+					title: 'Select a Folder to setup as test',
+				});
+				if (!folders || folders.length !== 1) {
+					return;
+				}
+				const folder = folders[0].path.replace(projectDir + '/', '');
+				await this.proxyToTaqAndShowOutput(
+					`test --init ${folder}`,
+					{
+						finishedTitle: `Setup folder ${folder} as test`,
+						progressTitle: `Setting up folder ${folder} as test`,
+					},
+					projectDir,
+					false,
+				);
+			},
+		);
+	}
+
+	exposeRunTestCommand(projectDir?: Util.PathToDir) {
```

Comment:
The parameter `projectDir` is optional but the body of this function uses `projectDir` without checking if it's provided.